h2o-core/src/main/java/water/api/RapidsHandler.java

```diff
@@ -59,18 +64,39 @@ public RapidsSchemaV3 exec(int version, RapidsSchemaV3 rapids) {
     }
   }
 
+  private static Class[] getRapidsClasses(){
+    try {
+      InputStream is = Thread.currentThread().getContextClassLoader().getResourceAsStream("rapids.list");
```

Comment:
@mmalohlava please see the comment above. I think the SPI can't be used here as per the reasons above without changing the semantics. If that would be acceptable, I propose to not include abstract classes into the output of rapids help. But if you have some tips how to use it, pls let me know! 

Good point regarding the class loaders! 👍